src/pretix/static/pretixpresale/js/widget/widget.js

```diff
@@ -1904,6 +1910,7 @@ var create_widget = function (element) {
     var items = element.attributes.items ? element.attributes.items.value : null;
     var variations = element.attributes.variations ? element.attributes.variations.value : null;
     var categories = element.attributes.categories ? element.attributes.categories.value : null;
+    var single_item_select = element.getAttribute("single-item-select") || "checkbox";
```

Comment:
A docs mention is missing, maybe even including a warning that the default might change in the future

Reply:
I added some docs with a note, that this default might change in the future. I also added a small screenshot to better illustrate what the docs are about.